packages/workers-shared/router-worker/src/index.ts

```diff
@@ -1,21 +1,17 @@
-import type { Fetcher, Request } from "@cloudflare/workers-types";
-
 interface Env {
 	ASSET_WORKER: Fetcher;
 	USER_WORKER: Fetcher;
 }
+
 export default {
 	async fetch(request: Request, env: Env) {
+		const maybeUserRequest = request.clone();
```

Comment:
I'll need to come back to this, but just as a reminder that I have a questions here... do we want to `clone()` here or create a new req via the `Request` constructor?